sdk/python/feast/config.py

```diff
@@ -113,24 +114,32 @@ def __init__(
         self._options = {}
         if options and isinstance(options, dict):
             self._options = options
+        self._defaults = opt().defaults()
 
         self._config = config  # type: ConfigParser
         self._path = path  # type: str
 
-    def get(self, option):
+    def get(self, option, default=_UNSET):
         """
         Returns a single configuration option as a string
 
         Args:
             option: Name of the option
+            default: Default value to return if option is not found
 
         Returns: String option that is returned
 
         """
+        default = {option: default} if default is not _UNSET else {}
         return self._config.get(
             CONFIG_FILE_SECTION,
             option,
-            vars={**_get_feast_env_vars(), **self._options},
+            vars={
```

Comment:
By the way, I think you should submit this change as part of a separate PR please.

Reply:
Created new PR, it seems  it needs to be merged before this one